Source/Core/InputCommon/ControllerEmu/ControlGroup/AnalogStick.cpp

```diff
@@ -56,14 +56,14 @@ ControlState AnalogStick::GetGateRadiusAtAngle(double ang) const
   return m_stick_gate->GetRadiusAtAngle(ang);
 }
 
-OctagonAnalogStick::OctagonAnalogStick(const char* name, ControlState gate_radius)
-    : OctagonAnalogStick(name, name, gate_radius)
+OctagonAnalogStick::OctagonAnalogStick(const char* _name, ControlState _gate_radius)
```

Comment:
We prefer to prefix names with the underscore after the identifier like so: `name_`

Ditto for all other occurrences

Reply:
I use this style because in ControlGroup subfolder, there is exiting code like this. If you prefer other way, I can submit a seperate patch to cleanup them.

Reply:
Why did you have to change this anyways? Members should have the `m_` prefix already and not cause any shadowing with the parameters.
In case I did miss some, use a suffix as lioncash suggested.

Reply:
sure. I can make a big change to add a prefix m_ to the fields in the class of ControlGroup

Reply:
Please don't. I only see `name` being used there, so I don't quite see the need to rename the others (like `gate_radius`) though?

Reply:
so only change name and ui_name here?

Reply:
Only the ones necessary to resolve the shadowing.